docs/events/Ability_Summit.md

```diff
@@ -0,0 +1,41 @@
+# ElectionGuard / E2E at the Microsoft Ability Summit May 10-12, 2022
```

Comment:
A note on this file name since it impacts the navigation would be to add the date. 

ability_summit_2022

Reply:
should we be lowercasing all other filenames too (e.g., Manifest_Building.md)?  i'll also be renaming this since we won't be part of the ability summit: should it be something like ```eg_e2e_event_june_2022.md```?

Reply:
We likely should at some point to choose that across all the files for consistency. 

But yes.